rd_ui/app/scripts/ng_highchart.js

```diff
@@ -325,6 +325,36 @@
                 series = seriesCopy;
               }
 
+              if (chartOptions['sortY'] === true || chartOptions['sortY'] === undefined) {
```

Comment:
`chartOptions['sortY'] === undefined` means that "old" queries without this setting will default to enabling this, right? I prefer that we don't do this, to avoid confusion and unexpected results for existing charts/users.